src/main/java/hudson/plugins/clover/CloverBuildWrapper.java

```diff
@@ -27,8 +27,10 @@
 import org.apache.commons.lang.StringUtils;
 import org.kohsuke.stapler.DataBoundConstructor;
 import org.kohsuke.stapler.StaplerRequest;
+import edu.umd.cs.findbugs.annotations.NonNull;
```

Comment:
Why did you change `javax.annotation` to `edu.umd.cs.findbugs.annotations`? The former one was JDK standard, the latter comes from a third party library.

Reply:
A comment in [JENKINS-55973](https://issues.jenkins.io/browse/JENKINS-55973) describes why JSR-305 has been replaced with spotbugs annotations in Jenkins core and in Jenkins plugins.  

The fundamental message (for me) was that JSR-305 was never completed.  It never became a part of the officially supported JDK.  Thankfully, Google and others have adopted spotbugs as the actively developed replacement for findbugs.  It is maintained and often releases new versions with additional capabilities.

Reply:
Thank you for clarification. Let's switch to findbugs-annotations then.